Move iOS, watchOS, and tvOS queues to latest releases

huskyroll opened the pull request:
#### 95507e3a1a4a919d1a156abbc279fdf6d24b13f5
```
Move iOS, watchOS, and tvOS queues to latest releases
https://bugs.webkit.org/show_bug.cgi?id=230837

Reviewed by NOBODY (OOPS!).

* CISupport/build-webkit-org/config.json:
* CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):
* CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.ios-simulator-14 img.logo): Deleted.
(table.queue-grid tr.platform.ios-14 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-simulator-14 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-14 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-simulator-7 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-7 img.logo): Deleted.
```

LionsAd commented:
Filed as https://bugs.webkit.org/show_bug.cgi?id=161780, so closing.

JonWBedard reviewed (approved):
Looks good! Needs to land via bugzilla still, though

huskyroll committed:
Move iOS, watchOS, and tvOS queues to latest releases
https://bugs.webkit.org/show_bug.cgi?id=230837

Reviewed by NOBODY (OOPS!).

* CISupport/build-webkit-org/config.json:
* CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):
* CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.ios-simulator-14 img.logo): Deleted.
(table.queue-grid tr.platform.ios-14 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-simulator-14 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-14 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-simulator-7 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-7 img.logo): Deleted.
